pywbem_mock/_wbemconnection_mock.py

```diff
@@ -225,31 +208,22 @@ class FakedWBEMConnection(WBEMConnection):
     generate responses in the same way the WBEM server would.
 
     For a description of the operation methods on this class, see
-    :class:`pywbem.WBEMConnection`.
+    :ref:`Faked WBEM operations`.
```

Comment:
Question: We don't have detailed operation method descriptions in the fake wbemconnection.  Shouldn't this point back to WBEMConnection??

By the way, It is harder and harder to keep all this in my context as I get further into the review so I may be wrong.

Reply:
**Discussion point:** We have the list of them in section "Faked WBEM operations", and that's where the reader needs to start reading because the descriptions there point out what the differences are, plus each description contains a link to the corresponding WBEMConnection method. By linking to WBEMConnection directly one misses the differences.